.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,20 @@
-<!--- Provide a general summary of your changes in the Title above -->
+<!--- Write a general summary of your changes in the Title field above -->
 
-## Description
-<!--- Describe your changes in detail -->
+<Addresses/Closes> #<issue/pr link>. <!---Describe the original problem or feature proposal, and briefly how this PR resolves it. All PRs should be opened against an issue.-->
```

Comment:
Can we start with something like "Before opening a PR, be sure there is a corresponding issue discussing this.  We can help you determine the best way to solve the problem before spending time on a solution that may not be accepted. Reference the issue number using a pound-sign before the number, like `#123`." or something like that. I know "All PRs should be opened against an issue" says essentially the same thing but that is pretty jargony. (Yeah I made up *jargony* what are you going to do about it?)